datatypes/src/raster/operations/map_indexed_elements.rs

```diff
@@ -448,11 +448,11 @@ where
 
         MaskedGrid::new(
             Grid::new(data.shape, out_data)
-                .expect("the shape of the grid and the data size matched before"),
```

Comment:
i think that the old reason is that we already did the same thing and it worked so it must work here. I think that is more information than "it shuld match". (same for all the other cases)